app/ante/eth.go

```diff
@@ -232,6 +235,14 @@ func (egcd EthGasConsumeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 	ctx = ctx.WithGasMeter(ethermint.NewInfiniteGasMeterWithLimit(gasWanted))
 	ctx.GasMeter().ConsumeGas(gasConsumed, "copy gas consumed")
 
+	// Add total gasWanted to cumulative in block transientStore for BaseFee calculation
+	if london {
```

Comment:
shouldn't this also check if the param is enabled? I think there is a util function available on the EVM module